src/DiagramGenerator/Generator.php

```diff
@@ -9,7 +9,7 @@
 use DiagramGenerator\Diagram\Board;
 use DiagramGenerator\Exception\UnsupportedConfigException;
 use Symfony\Component\Validator\Validation;
-use Symfony\Component\Validator\Validator;
+use Symfony\Component\Validator\Validator\LegacyValidator as Validator;
```

Comment:
does this exist in Symfony 2.3?

If we merge this, it would block current beta, since we couldn't update code on the library and push it ahead to take those changes, because this would error out.

If it doesn't exist, I think the Symfony upgrade branch composer.json can be directed to using branch of the library for now, unless the plan is to merge the Symfony upgrade right now, right after merging this, then disregard this comment.